bundles/org.openhab.binding.airgradient/README.md

```diff
@@ -83,6 +83,7 @@ For more information about the data in the channels, please refer to the models
 | atmp       | Number:Temperature   | Read       | Ambient Temperature                                                              |
 | rhum       | Number:Dimensionless | Read       | Relative Humidity Percentage                                                     |
 | wifi       | Number               | Read       | Received signal strength indicator                                               |
+| boot       | Number:Dimensionless | Read       | Number of measure uploads since last reboot (boot)                               |
```

Comment:
Why boot? Could be a biut more meaningfull, like `measure-upload-count` or `count-since-boot`

Reply:
I agree that the name is bad. It is a value which is used for troubleshooting (if it always stays low it means your sensor is rebooting a lot). My reasoning for using the "boot" name with a better description is that in the airgradient forums when people are troubleshooting, I see them ask for "what is the boot count". And therefore I thought it would make sense to use the same (misleading) name. I'll change channel to "uploads-since-reboot", but keep the "(boot)" in the description to refer to the airgradient lingo.